packages/insights/src/stats/vector.ts

```diff
@@ -8,29 +8,39 @@ export const NUMBER_OF_BUCKETS = 50;
  * result in the last bucket.
  */
 export const MAX_VALUE = 10000;
+export const TIMELINE_MAX_VALUE = 60000 * 15;
```

Comment:
can you change this to make it easier to read please?

```suggestion
export const TIMELINE_MAX_VALUE = 15 * 60 * 1000;
```